lua/telescope/builtin/internal.lua

```diff
@@ -334,18 +334,51 @@ end
 
 internal.help_tags = function(opts)
   local all_tag_files = {}
+  local all_tag_lang_files = {}
   local all_help_files = {}
-  for _, v in ipairs(vim.split(vim.fn.globpath(vim.o.runtimepath, 'doc/*', 1), '\n')) do
+  local langs = {}
+  for _, v in ipairs(vim.fn.globpath(vim.o.runtimepath, 'doc/*', 1, 1)) do
     local split_path = vim.split(v, path.separator, true)
     local filename = split_path[#split_path]
     if filename == 'tags' then
       table.insert(all_tag_files, v)
+    elseif filename:match('^tags%-..$') then
```

Comment:
Does the file always has a 2 char width lang code or are there exceptions?
Because i most definitely see exceptions in `/etc/locale.gen` like some have 3

Reply:
[Neovim doc][] says the extension of help files has `.XXx` format and tags have `tags-XX` filename. The lang `XX` has two letters. I see the 3rd and later letters are cut in the source.

[Neovim doc]: https://neovim.io/doc/user/helphelp.html#help-translated

https://github.com/neovim/neovim/blob/77a6049e07d070470d3274348ac61ab9b96cac6f/src/nvim/option.c#L874